lib/dotcom_web/controllers/cms_controller.ex

```diff
@@ -137,6 +138,17 @@ defmodule DotcomWeb.CMSController do
     end
   end
 
+  defp render_page(conn, %Page.Diversions{} = page) do
+    conn
+    |> assign(
+      :alerts,
+      Alerts.Repo.diversions_by_route_ids(page.related_transit, conn.assigns.date_time)
```

Comment:
📓 If we're rendering a diversion page then we get diversions for any route that the page is tagged with. That's how we know that this diversion page is for the Red Line, for example.